frame/support/src/traits/misc.rs

```diff
@@ -269,3 +269,17 @@ pub trait OffchainWorker<BlockNumber> {
 	fn offchain_worker(_n: BlockNumber) {}
 }
 
+/// Some amount of backing from a group. The precise defintion of what it means to "back" something
+/// is left flexible.
+pub struct Backing {
+	/// The number of members of the group that back some motion.
+	pub approvals: u32,
+	/// The number of members of the group that do not back the motion.
+	pub eligible: u32,
```

Comment:
1. If the doc is correct, I'd expect this to be called `rejection`, and all eligible would be `approvals + rejections`.

```suggestion
	/// The number of members of the group that do not back the motion.
	pub rejections: u32,
```

2. If _eligible_ is correct then the doc is misleading:

```suggestion
	/// The total count of group members.
	pub eligible: u32,
```

3. or this is intended, and I am unfamiliar with the jargon

Reply:
should be (2) i think